builders/mariadb.js

```diff
@@ -11,22 +11,22 @@ module.exports = {
     version: '10.3',
     supported: ['11.3', '11.2', '11.1', '11.0', '10.11', '10.10', '10.9',
       '10.8', '10.7', '10.6', '10.5', '10.4', '10.3', '10.2', '10.1'],
-    legacy: ['10.10', '10.9', '10.8', '10.7', '10.3', '10.2', '10.1'],
+    legacy: ['11.3', '10.10', '10.9', '10.8', '10.7', '10.3', '10.2', '10.1'],
```

Comment:
Wouldn't legacy be 11.0, 11.1 and 11.2?

Reply:
Oh, that isn't supposed to be there. 11.0 will become EOL in a few weeks.

I looked at https://endoflife.date/mariadb listing, which incorrectly states 11.3 as EOL. It's a rolling release (with no set EOL date?).